### PR TITLE
Wrote basic API routes

### PR DESCRIPTION
**Pull request by ShepLT1**

### CODE REVIEW

File: client/src/index.js [hunk 1]
```diff
@@ -1,17 +1,5 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
 import App from "./App";
-import reportWebVitals from "./reportWebVitals";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
```
Comment (zapponejosh) [inline client/src/index.js:10]: @ShepLT1 is there a reason you removed strictMode from this file? seems like we should leave it in.

File: client/src/pages/User.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    API.updateChall()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+  };
+
+  loadUserRoutes()
+
+
```
Comment (zapponejosh) [inline client/src/pages/User.js:63]: @ShepLT1 can you put comments in for each route? (what they will take in and what they return)

Comment (zapponejosh) [reply client/src/pages/User.js:63]: actually, why do you need to load all the routes here? why not just keep them in the API until they are needed?

File: client/src/utils/API.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  getPhysActSugg: function (id) {
+    return axios.get("/api/dashboard/" + id + "/physical");
+  },
+  getMentalActSugg: function (id) {
+    return axios.get("/api/dashboard/" + id + "/mental")
+  },
+  updateChall: function (id) {
+    return axios.put("/api/dashboard/" + id);
+  }
```
Comment (zapponejosh) [inline client/src/utils/API.js:37]: Maybe those comments should go in this file actually

File: server/routes/api/dashboard.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  .get(controller.getMovie)
+
+router
+  .route("/:id/physical")
+  .get(controller.getPhysAct)
+
+router
+  .route("/:id/mental")
+  .get(controller.getMentalAct)
+
```
Comment (zapponejosh) [inline server/routes/api/dashboard.js:25]: @ShepLT1 can you add comments for where these should be used/ by which component on the frontend?

**Review by zapponejosh — CHANGES_REQUESTED** [review]
Hey! take a look at my comments. mostly just asking to add comments in so we know where things go and what the expected behavior is as we build the app... also get rid of that ds_store file!!

File: client/src/pages/User.js (hunk 2 above)
Comment (ShepLT1) [reply client/src/pages/User.js:63]: just to check that they were working. I can take it out now

Comment (zapponejosh) [reply client/src/pages/User.js:57]: You can leave them in commented out with an explanation too if you think they are good to have for testing

File: client/src/utils/API.js (hunk 3 above)
Comment (zapponejosh) [reply client/src/utils/API.js:44]: looks good. we may not need to use id in the URL path if we can access the user object set using passport

**Review by zapponejosh — APPROVED** [review]

**Pull request merged**